systemtest/tests/integration/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -ux
+
+# get to project root
+cd ../../../
+dnf --setopt install_weak_deps=False install -y \
+  podman git-core python3-pip python3-pytest rhc-worker-playbook yggdrasil
+
+podman run --rm --detach --publish 1883:1883 --name mosquitto \
+--security-opt  label=disable --volume \
+"./integration-tests/resources/mosquitto_config/mosquitto.conf:/mosquitto/config/mosquitto.conf" \
+docker.io/eclipse-mosquitto
```

Comment:
This is purely cosmetic, but I find lines like this easier to read when they are indented.
```suggestion
podman run --rm --detach --publish 1883:1883 --name mosquitto \
    --security-opt  label=disable --volume \
    "./integration-tests/resources/mosquitto_config/mosquitto.conf:/mosquitto/config/mosquitto.conf" \
    docker.io/eclipse-mosquitto
```